app-containers/docker/autobuild/postinst

```diff
@@ -7,5 +7,5 @@ fi
 
 # Remove /etc/conf.d/docker 
 if dpkg-maintscript-helper supports rm_conffile 2>/dev/null; then
-    dpkg-maintscript-helper rm_conffile /etc/conf.d/docker 27.3.1+tini0.19.0~ -- "$@"
+    dpkg-maintscript-helper rm_conffile /etc/conf.d/docker 27.3.1+tini0.19.0-1~ -- "$@"
```

Comment:
Hardcoding version number in the scripts might not be a good idea... Consider adding hints for other developers in spec file?

Reply:
Adding a comment would suffice, this is the version where `/etc/conf.d/docker` was last shipped.